products/green_fast_track/models/staging/stg__nys_parks_properties.sql

```diff
@@ -0,0 +1,17 @@
+-- stg__nys_parks_properties.sql
+
+WITH source AS (
+    SELECT *
+    FROM {{ source('recipe_sources', 'nysparks_parks_polygons') }}
+),
+
+final AS (
+    SELECT
+        'nys_parks_properties' AS variable_type,
+        objectid AS variable_id,
+        st_transform(wkb_geometry, 2263) AS raw_geom
+    FROM source
+    WHERE upper(county) IN ('BRONX', 'KINGS', 'QUEENS', 'RICHMOND')
```

Comment:
missing `MANHATTAN`, I'm seeing two records with that county value

Reply:
In the GDE notes for this variable in the excel sheet, it is the following:
> Filter down to NYC, Use County field and select NYC Counties: Bronx, Kings, Queens, Richmond (none Manhattan - New York)

I understood it as not to include Manhattan. Does that mean the opposite?

@damonmcc

Reply:
oh sorry I didn't see that! seems odd to exclude Manhattan but the excel sheet says to, so never mind my comment!